styles.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@media only screen and (min-width: 375px) and (max-width: 767px) {
+  header {
+    position: relative;
+    z-index: 1;
+    width: 100%;
+  }
+
+  body {
+    background-color: #1c1a19;
+    background-image: url(./img/Illustration.png);
+    background-repeat: no-repeat;
+    background-position: 100% 210%;
+  }
+
+  footer {
+    display: flex;
+  }
+
+  .toolbar {
+    display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.
- Kindly note that DRY is a requirement, but since the other